src/agent/Core/ApplicationPool/Options.h

```diff
@@ -211,6 +211,11 @@ class Options {
 	 * on spawning the process or the preloader. */
 	unsigned int startTimeout;
 
+	/** The maximum amount of time, in milliseconds, that may be spent
+	 * waiting on the process to accept controller's connection on the
+	 * socket. */
+	unsigned int connectTimeout;
```

Comment:
No need to propagate this to ApplicationPool. We only use this option in the Controller.